client/src/App/App.tsx

```diff
@@ -216,6 +228,48 @@ const App: React.FC = () => {
               path={`${URL_USER_CREDENTIALS}/:${ROUTE_PARAM_USER_ID}`}
               component={ConnectedUserCredentials}
             />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={false}
```

Comment:
Let's use the env variable being used instead of the value `false`

Reply:
Addressed